demos/Phaster.ipynb

```diff
@@ -65,18 +64,14 @@
     "    for line in f:\n",
     "        acc = line.rstrip()\n",
     "        #print(acc)\n",
-    "        apiurl = \"http://phaster.ca/phaster_api?acc={}\".format(acc) \n",
-    "        r=requests.get(apiurl)\n",
-    "        try:\n",
-    "            d=r.json()['summary']\n",
-    "        except ValueError:  # includes simplejson.decoder.JSONDecodeError\n",
-    "            # wait a delay and try again\n",
-    "            time.sleep(2)\n",
-    "            try:\n",
-    "                r=requests.get(apiurl)\n",
-    "                d=r.json()['summary']\n",
-    "            except ValueError:  # includes simplejson.decoder.JSONDecodeError\n",
-    "                print(\"Error acquiring results for accession %s\" %acc)\n",
+    "        apiurl = \"http://phaster.ca/phaster_api?acc={}\".format(acc)\n",
+    "        \n",
+    "        def apicall():\n",
```

Comment:
It might be a good idea to define `apicall()` outside the for loop.

Reply:
@aays , when I used it earlier, everything went trough. Now, I see the same error. I tried to fix it, but I have yet found the fix....